tests/test_gzip_compliance.py

```diff
@@ -804,6 +804,19 @@ def test_decompress_stdin_stdout(self):
         self.assertEqual(err, b'')
         self.assertEqual(out, self.data)
 
+    # The following tests use assert_python_failure or assert_python_ok.
+    #
+    # If the env_vars argument to assert_python_failure or assert_python_ok
+    # is empty the test will run in isolated mode (-I) which means that the
+    # PYTHONPATH environment variable will be ignored and the test fails
+    # because the isal module can not be found, or the test is run usung the
```

Comment:
There is a typo here. "usung" → "using".

Reply:
Fixed: https://github.com/pycompression/python-zlib-ng/commit/34c171bc51ce1c7dcb2e1e8cbb79a903afd7bd90